rules/integrations/aws/defense_evasion_cloudtrail_logging_deleted.toml

```diff
@@ -86,9 +88,26 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:aws.cloudtrail and event.provider:cloudtrail.amazonaws.com and event.action:DeleteTrail and event.outcome:success
+event.dataset:aws.cloudtrail
+    and event.provider:cloudtrail.amazonaws.com
+    and event.action:DeleteTrail
+    and event.outcome:success
 '''
 
+[rule.investigation_fields]
+field_names = [
+    "event.action",
+    "event.outcome",
```

Comment:
```suggestion
```
event.outcome in the rule is already set to "success" so not sure it's a useful field to show for this alert

Reply:
That is a valid point. I questioned included these as well, but it came down to the purpose of them, which is to pivot into timeline from the key:value pairs that you can add from the highlighted fields. Thus if a user wanted to ask "for this user (add to timeline) for this outcome (add to timeline) show me what else..." it would be available. If we think that is unlikely to happen, happy to remove.

Reply:
Mmm I see your point, I'd keep it then for those reasons.